packages/mermaid/src/diagrams/sankey/sankeyDiagram.ts

```diff
@@ -0,0 +1,13 @@
+import { DiagramDefinition } from '../../diagram-api/types.js';
+// @ts-ignore: TODO Fix ts errors
+import parser from './parser/sankey.jison';
```

Comment:
No need to mark jison errors as TODO. We cannot fix them.

Reply:
This ts-ignore works for the whole file, I suppose, so we need like an option to ignore only one line...

Reply:
This is a `@ts-ignore`, which only skips next line. `@ts-nocheck` is the one which skips the file. 

Changing the comment to `// @ts-ignore: JISON doesn't have type support`, should be enough.

Reply:
That what I'm currently using:

https://github.com/mermaid-js/mermaid/blob/b041e5168bc6bcdef4be6d443b6774f89b55a782/packages/mermaid/src/diagrams/info/infoDiagram.ts#L2

Reply:
Thanks! That's great

Reply:
Fixed